### ISSUE TITLE
Rename Wired Iot Network on the ER-X Dashboard?

### ISSUE THREAD

**Issue by dr4ugr** [mjp66/Ubiquiti#59]
Hi! I was making some updates to the config on my home network and I noticed that you don't mention updating the name of Wired Iot Network to something else (see pages 158-159). I hope I didn't miss anything but is it safe to update that in the dashboard to HomeNet?

**Comment by mjp66** [comment]
Hi,

Sorry, but I am having problems parsing / understanding your posting.
I know that this guide can be confusing, as to its age, many revisions, and the fact that I finally learned VLANs late in the game.

> I hope I didn't miss anything but is it safe to update that in the dashboard to HomeNet?
I read "that" in your posting as renaming the IOT Network to the HomeNet Network. Which doesn't make any sense.

So I will just provide section references about renaming Networks. References per 12/19/2021 revision of the guide.

15. Network Naming
32. Set Domain Names for Networks
34. Rename DHCP Servers
63. EdgeMax Change Interface Names
90. Coalescing the Wired Iot and Wi-Fi Iot Networks
    23 - dnsmasq
    29 - DNS Forwarding

Hope this helps,
-Mike

**Comment by dr4ugr** [comment]
Thank you for the response. I wasn't sure if this is the right forum to do this. If it's not, I can cease.

I followed all of these steps to the letter and just recently added another Unifi device. I then went through 90 Coalescing...but I still have a network called Wired IoT Network in my Dashboard. I wasn't sure if I was supposed to keep that as is or rename it to (and do the steps to add Separate2) to Separate3.

Or, did I miss the point of the exercise and it should remain an IoT network
